test/new-e2e/tests/updater/linux_test.go

```diff
@@ -34,13 +35,28 @@ const (
 
 type vmUpdaterSuite struct {
 	e2e.BaseSuite[environments.Host]
+	packageManager string
 }
 
-func TestUpdaterSuite(t *testing.T) {
-	e2e.Run(t, &vmUpdaterSuite{}, e2e.WithProvisioner(awshost.ProvisionerNoFakeIntake(
+func runTest(t *testing.T, pkgManager string, arch os.Architecture, distro os.Descriptor) {
+	reg := regexp.MustCompile(`[^a-zA-Z0-9_\-.]`)
+	testName := reg.ReplaceAllString(distro.String()+"-"+string(arch), "_")
+	e2e.Run(t, &vmUpdaterSuite{packageManager: pkgManager}, e2e.WithProvisioner(awshost.ProvisionerNoFakeIntake(
 		awshost.WithUpdater(),
-		awshost.WithEC2InstanceOptions(ec2.WithOSArch(os.UbuntuDefault, os.ARM64Arch)),
-	)))
+		awshost.WithEC2InstanceOptions(ec2.WithOSArch(distro, arch)),
+	)),
+		e2e.WithStackName(testName),
+	)
+}
+
+func TestCentOS(t *testing.T) {
+	t.Parallel()
+	runTest(t, "rpm", os.AMD64Arch, os.CentOSDefault)
```

Comment:
<!-- Warning highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes and they are blockers. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question. -->
**🔨 warning**
There is a risk of OOM for the runner, we suggest running one job per Test per remote instance

Reply:
I'll update on a next PR where I'm adding more cases